nannou/src/draw/renderer/mod.rs

```diff
@@ -395,7 +399,7 @@ impl Renderer {
             .usage(wgpu::TextureUsage::SAMPLED | wgpu::TextureUsage::COPY_DST)
             .format(Self::GLYPH_CACHE_TEXTURE_FORMAT)
             .build(device);
-        let glyph_cache_texture_view = glyph_cache_texture.create_default_view();
+        let glyph_cache_texture_view = glyph_cache_texture.view().build();
```

Comment:
The equivalent in wgpu 0.6 to `create_default_view()` is `create_view(&wgpu::TextureViewDescriptor::default())`.

Reply:
[1cf9fce](https://github.com/nannou-org/nannou/pull/655/commits/1cf9fce293cffbfdd92d50e18db80f674dfcc4ae)